src/terraform_drift_detection/terraform_drift_detection/init.py

```diff
@@ -0,0 +1,32 @@
+import logging
+import re
+
+from terraform_drift_detection.config import config
```

Comment:
This import statement instantiates [the `config` object in `config.py`](https://github.com/artsy/opstools/pull/12/files#diff-245f605e2e7b46b90e532b35a062f440b738c4533d3111c3665cc11f8754775aR34). Is this good practice? It feels too implicit. Wonder if it's better to import a function that does the instantiation, and call that function, which would be more explicit. I guess that function can do the instantiation only if `config` object is `None`.